src/backend/lib.rs

```diff
@@ -743,6 +733,23 @@ fn user() {
             .unwrap_or_default();
         if own_profile_fetch {
             user.accounting.clear();
+        } else {
+            let karma = reaction_karma();
+            user.endorsements = user
+                .posts
+                .iter()
+                .rev()
+                .take(CONFIG.feed_page_size * 3)
+                .filter_map(|post_id| state.posts.get(post_id))
+                .flat_map(|post| post.reactions.iter())
+                .flat_map(|(r_id, users)| {
+                    let cost = karma.get(r_id).copied().unwrap_or_default();
+                    users.iter().map(move |user_id| (*user_id, cost))
+                })
+                .fold(BTreeMap::default(), |mut acc, (user_id, karma)| {
+                    acc.entry(user_id).and_modify(|e| *e += karma).or_insert(1);
```

Comment:
I think this is a bug
```suggestion
                    acc.entry(user_id).and_modify(|e| *e += karma).or_insert(karma);
```

Reply:
Oh, thanks so much! I'm so glad we have the reviews now. In my initial implementation I only counted the endorsers. Then I decided to count the actual karma and forgot to update this line. Thank you!!!